app/app.go

```diff
@@ -304,32 +309,34 @@ func NewOsmosisApp(
 	app.CrisisKeeper = crisiskeeper.NewKeeper(
 		app.GetSubspace(crisistypes.ModuleName), invCheckPeriod, app.BankKeeper, authtypes.FeeCollectorName,
 	)
-	app.UpgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath)
 
-	// this configures a no-op upgrade handler for the v2 upgrade,
-	// which improves the lockup module's store management.
-	app.UpgradeKeeper.SetUpgradeHandler(
-		"v4", func(ctx sdk.Context, plan upgradetypes.Plan) {
-			// Upgrade all of the lock storages
-			locks, err := app.LockupKeeper.GetLegacyPeriodLocks(ctx)
-			if err != nil {
-				panic(err)
-			}
-			// clear all lockup module locking / unlocking queue items
-			app.LockupKeeper.ClearAllLockRefKeys(ctx)
-			app.LockupKeeper.ClearAllAccumulationStores(ctx)
+	/*
+		// this configures a no-op upgrade handler for the v2 upgrade,
+		// which improves the lockup module's store management.
+		app.UpgradeKeeper.SetUpgradeHandler(
+			"v4", func(ctx sdk.Context, plan upgradetypes.Plan) {
+				// Upgrade all of the lock storages
+				locks, err := app.LockupKeeper.GetLegacyPeriodLocks(ctx)
+				if err != nil {
+					panic(err)
+				}
+				// clear all lockup module locking / unlocking queue items
+				app.LockupKeeper.ClearAllLockRefKeys(ctx)
+				app.LockupKeeper.ClearAllAccumulationStores(ctx)
 
-			// reset all lock and references
-			if err := app.LockupKeeper.ResetAllLocks(ctx, locks); err != nil {
-				panic(err)
-			}
+				// reset all lock and references
+				if err := app.LockupKeeper.ResetAllLocks(ctx, locks); err != nil {
+					panic(err)
+				}
```

Comment:
I think we should un-comment this part out. I would like all handlers to still be defined personally. (Though maybe we move it to a new file and comment out the actual code.)

With the handlers, we can do things like https://github.com/cosmos/cosmos-sdk/pull/10407